react-app/src/native/components/SportChooser.js

```diff
@@ -0,0 +1,137 @@
+import React from "react";
+import { View, Image, StyleSheet, TouchableHighlight, Button, Text} from "react-native";
+import { Container, Content, Card } from 'native-base'
+
+export default function PickSport({route, navigation}) {
+
+    let sport
+
+    return (
+        <Container>
+            <Content>
+                <Card>
+                    <View style={{
+                        flex: 1,
+                        flexDirection: 'column',
+                        justifyContent: 'space-between',
+                        alignItems:'center'
+                    }}>
+                        <Text style={styles.header}>Choose a 
+                            <Text style={{color:'red'}}> sport</Text>
```

Comment:
I haven't run the app yet but i'm 99.9% sure this would look better if you used Colors.brandSecondary (see /components/common/Loadin.js for how to use common colors from the predefined theme)